src/protocols/rdp/keymaps/fr_ch_qwertz.keymap

```diff
@@ -0,0 +1,59 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+parent  "base"
+name    "fr-ch-qwertz"
+freerdp "KBD_SWISS_FRENCH"
+
+#
+# Basic keys
+#
+
+map -altgr -shift 0x29 0x02..0x0C      ~ "§1234567890'"
+map -altgr -shift      0x10..0x1A      ~ "qwertzuiopè"
+map -altgr -shift      0x1E..0x28 0x2B ~ "asdfghjkléà$"
+map -altgr -shift 0x56 0x2C..0x35      ~ "<yxcvbnm,.-"
+
+map -altgr +shift 0x29 0x02..0x0C      ~ "°+"*ç%&/()=?"
+map -altgr +shift      0x10..0x1B      ~ "QWERTZUIOPü!"
+map -altgr +shift      0x1E..0x28 0x2B ~ "ASDFGHJKLöä£"
+map -altgr +shift 0x56 0x2C..0x35      ~ ">YXCVBNM;:_"
+
+#
+# Keys requiring AltGr
+#
+
+map +altgr -shift 0x02..0x04 ~ "¦@#"
+map +altgr -shift 0x06..0x08 ~ "¬|¢"
```

Comment:
0x06..0x08 is wrong... look for example at https://upload.wikimedia.org/wikipedia/commons/thumb/4/4f/KB_Swiss.svg/900px-KB_Swiss.svg.png. You see that there are two keys between the hash and the next alt-gr key to the right. So if you skip 0x05 and 0x06, the next key you get is 0x07.